### ISSUE TITLE
Accessibility improvements: Headings, Landmarks, etc.

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#820]
This will be an inventory ticket for some broad (and some detailed) accessibility improvements. The overall goal is to ensure that:
- there is a meaningful heading structure that communicates the structure and focus of the page; with no missing heading levels
- there is a meaningful landmark structure that both passes audits (does not currently) and communicates the structure of the page
- svgs/icons are perceivable but don't get in the way
- some hidden pre-BS4 workarounds are updated or removed so they don't cause confusion.

Some of these are Blacklight-level changes, but they should be uncontroversial.

## Headings
- [x] Change `<h1>Archival Collections at Institution</h1>` to a `<span>` on all but the home page. #821
- [x] Add a page-specific `<h1>` to all page templates #834 
- [x] Make the `<title>` content match (or close to) the `<h1>` for the page #842
- [x] Update heading levels on the Repository page to eliminate missed levels #835 
- [x] Suppress "# collections" text on the Repository pages #836
- [x] Add/update headings in the tab sections on the Collection/Component pages #837
- [ ] Address the **`<h3>` everywhere** issue with component titles in hierarchy views

## Landmarks & roles
- [x] Make `<div id="main-container">` the `main` landmark #838
- [x] Add aria-labels to `<section>`s #841
- [x] Make the breadcrumb a labelled `<nav>` #839
- [ ] Add an aria-label to the search form [blacklight#2147](https://github.com/projectblacklight/blacklight/issues/2147)

## Jump-to links
- [ ] Add jump links to all pages #840

## svgs
- [ ] Make svg icons perceivable by screen readers #866
- [ ] Tooltips to be addressed at some future point

## Cleanup
- [ ] Move unassociated `sr-only` text to `aria-label` on results view button group
- [x] Hide pseudo-content (», others?) from SR. #854
- [x] Restore label to Bookmark checkboxes #862

more to come

**Comment by seanaery** [comment]
Resolving this issue as part of a review of `arclight 1.0.x`. As of v1, aXe DevTools no longer reports any issues related to headings and landmarks. Any specific accessibility issues that remain should be submitted as individual tickets (if they haven't already been).
